index.js

```diff
@@ -942,7 +977,7 @@
   //. > S.init([])
   //. Nothing()
   //. ```
-  S.init = R.ifElse(R.isEmpty, Nothing, R.compose(Just, R.init));
+  S.init = S.slice(0, -1);
```

Comment:
Actually, funnily enough, the thing I wasted most time on (aside from the whole enterprise, that is) was trying to get all the tests running only to finally realise that it was because `compose` is not autocurried.